packages/cosmic-swingset/test/prometheus.test.ts

```diff
@@ -157,17 +196,19 @@ avaRetry(test, 'Prometheus metric definitions', async t => {
       `slog metrics must include ${q(nameAndLabels)}`,
     );
     comparisonCount += 1;
-    const [fuzziness, unit] = (() => {
+    const [fuzziness, unit, ignorable] = (() => {
       if (fuzzinessOverrides.has(name)) return fuzzinessOverrides.get(name);
 
-      // Default to 100 milliseconds of wiggle room for timing data.
+      // Default to 100 milliseconds of wiggle room for timing data, but expand
+      // that to 500 under subprocess retry (where the greater load can widen
+      // discrepancies).
```

Comment:
Why is the load greater in a retry?

Reply:
I suspect it's because of AVA's heartbeat IPC.